### PR TITLE
Adds CTRL+Mouse wheel zoom

### PR DESCRIPTION
**Pull request by rhiroshi**
Adds a hotkey for zooming just like browsers.
Pressing CTRL and scrolling the mouse whell will zoom in or out.

### CODE REVIEW

File: app/editor/boot.js [hunk 1]
```diff
@@ -792,6 +792,17 @@ thin.init_ = function() {
       }
     });
 
+    // Mouse wheel zoom
+    document.body.addEventListener('mousewheel', function(e){
```
Comment (hidakatsuya) [inline app/editor/boot.js:796]: Although Ctrl+mouse wheel zoom events can also occur in the menu bar and on the right pane, I think it's more natural to trigger them only on the canvas. Could you change it as follows?

```suggestion
    main.getMain().getElement().addEventListener('mousewheel', function(e) {
```

File: app/editor/boot.js [hunk 2]
```diff
@@ -792,6 +792,17 @@ thin.init_ = function() {
...
+      var workspace = thin.core.getActiveWorkspace();
+      if(e.ctrlKey != true || !workspace) return;
```
Comment (hidakatsuya) [inline app/editor/boot.js:798]:
```suggestion
      if (!e.ctrlKey || !workspace) return;
```

File: app/editor/boot.js [hunk 3]
```diff
@@ -792,6 +792,17 @@ thin.init_ = function() {
...
+      if(e.wheelDeltaY < 0) {
+        workspace.getAction().actionSetZoom(workspace.getUiStatusForZoom() - 10);
+      }else {
+        workspace.getAction().actionSetZoom(workspace.getUiStatusForZoom() + 10);
+      }
+    })
```
Comment (hidakatsuya) [inline app/editor/boot.js:804]: I'd like you to modify the code style as follows, including other similar parts.
```suggestion
      if (e.wheelDeltaY < 0) {
        workspace.getAction().actionSetZoom(workspace.getUiStatusForZoom() - 10);
      } else {
        workspace.getAction().actionSetZoom(workspace.getUiStatusForZoom() + 10);
      }
    });
```

**Review by hidakatsuya — COMMENTED** [review]
Thank you for your contribution! Looks good, but i'd like some fixes.

**Comment by rhiroshi** [comment]
Sure! I accepted the suggested changes and checked for other changes to adapt to the code style and found none.

**Review by hidakatsuya — APPROVED** [review]

**Pull request merged**